3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1830,50 +1830,76 @@ Per avere traccia degli elementi verificati che costituiscono il prodotto, su Gi
 
 
 == Processo di Transizione <processo_transizione>
-Lo scopo del processo di Transizione è di stabilire la capacità di un sistema di fornire servizi specificati dalle richieste del Proponente all'interno di un'ambiente operativo differente da quello di sviluppo.
+
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.4.10_
+
+=== Scopo
+
+Il processo di Transizione ha lo scopo di stabilire la capacità del sistema di fornire i servizi specificati dalle richieste del Proponente (descritti in #adr_v ), all'interno di un ambiente operativo.
+
+Coinvolge l'installazione del sistema verificato al fine di consentire l'operatività in ambienti diversi, come quelli di sviluppo, test o rilascio in una diversa infrastruttura.
+
 
 === Risultati
-Come risultato della corretta applicazione del processo di transizione, otteniamo che:
 
-- il #man viene redatto;
-- gli utenti vengono supportati e formati all'utilizzo del sistema;
-- viene rilasciata una versione del sistema;
-- viene effettuato il deploy su Azure.
+I risultati della corretta applicazione del processo di transizione sono:
+
+- avviene il rilascio di una versione del sistema;
+- viene effettuato il deploy del software su Azure;
+- viene garantito il funzionamento dei servizi e l'operatività del sistema;
+- vengono identificati i risultati e le anomalie della transizione;
+- viene stabilita la tracciabilità degli elementi trasferiti;
+- viene redatto del documento #man;
+- viene fornito supporto e formazione degli utenti all'utilizzo del sistema.
 
 === Attività
-Questo processo è composto da varie attività indipendenti e di diversa natura.
 
-Le seguenti attività non sono quindi necessariamente consequenziali, ma la loro corretta applicazione porta al conseguimento dei risultati precedentemente descritti.
+==== Rilascio dell'ultima versione di sistema
+
+I rilasci del sistema avvengono esclusivamente quando esso viene ritenuto sufficientemente maturo e pronto per l'utilizzo e sono avvenuti sostanziali cambiamenti rispetto alla precedente versione rilasciata, come avanzamenti significativi o risoluzione di bug critici.
+
+Il repository GitHub dedicato a WMS3 è provvisto di un branch principale `dev`, il quale rappresenta la fonte da cui vengono creati i feature branch.
+Questi ultimi sono necessari al conseguimento degli obiettivi di un task e destinazione dei merge a seguito delle successive pull request.
+
+I rilasci del prodotto avvengono sempre dal branch di partenza `dev`, in quanto il codice contenuto in esso garantisce il soddisfacimento dei requisiti di qualità imposti dal gruppo.
+
+Ogni release avvenuta con successo, in conformità con il superamento dei test e le procedure descritte nei processi di implementazione (@processo_implementazione), integrazione (@processo_integrazione) e verifica (@processo_verifica), genera una release specifica tracciata e consultabile (@tracciabilità_risultati_release) con relativo numero di release aggiornato.
```

Comment:
Cambierei alla fine "numero di release" in "numero di versione" e taggerei la sezione 4.5.3.1.1.2, che descrive proprio il versionamento del software (non so se abbia già il tag)